RachealN/Web-Practical
# #167944176 I was able to add CSS styles to the Web API User Interface for good UI designs and user Experience

What does this PR do?
Add CSS styles to webform homepage user interface(UI) for good user experience

Description of Task to be completed?
- Added styles to the index.html(home page) with CSS

How should this be manually tested?
- git clone this repo: https://github.com/RachealN/Web-Practical.git on your local,
- cd to Web-Practical directory and open index.html page in any browser of your choice to see the Web API User Interface
- and go to UI/styles.css to check CSS styles

Any background context you want to provide?
N/A

What are the relevant pivotal tracker stories?
##167944176

Screenshots (if appropriate)
home page(UI)
![web UI](https://user-images.githubusercontent.com/41440727/63206428-ed5a1800-c0bc-11e9-906c-b7788f3726f6.PNG)

html code
![html](https://user-images.githubusercontent.com/41440727/63206441-17133f00-c0bd-11e9-8fc9-af49b16528cb.PNG)

CSS code
![Css](https://user-images.githubusercontent.com/41440727/63206445-28f4e200-c0bd-11e9-8833-6b2848ab8748.PNG)
